src/components/MaDropdown/MaDropdown.vue

```diff
@@ -1,43 +1,251 @@
+                                </template>
+                                <ma-button variant="link" :icon="extraIcon.icon"/>
+                            </a-tooltip>
+                            <ma-button v-else variant="link" :icon="extraIcon.icon"/>
+                        </div>
+                        <div v-if="buttonGroup.length > 0">
+                            <div
+                                v-for="(button, index) in buttonGroup"
+                                :key="index"
+                                class="flex flex-row mr-1"
+                            >
+                                <a-tooltip v-if="button.tooltip">
+                                    <template slot="title">
+                                        {{ button.tooltip }}
+                                    </template>
+                                    <ma-button
+                                        variant="link"
+                                        :icon="button.icon"
+                                        @click="$emit('buttonGroupEvent', $event)"
+                                    />
+                                </a-tooltip>
+                                <ma-button
+                                    v-else variant="link"
+                                    :icon="button.icon"
+                                    @click="$emit('buttonGroupEvent', $event)"
+                                />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </a-menu-item>
+            <!--there is a menu item if a button required in bottom of menu -->
+            <a-menu-item
+                v-if="menuItemButton"
+                class="ma-menu-item"
+                @click="$emit('menuItemClick', $event);"
+            >
+                <ma-icon v-if="menuItemButton.icon" :icon="menuItemButton.icon"/>
+                {{ menuItemButton.name }}
+            </a-menu-item>
+        </a-menu>
     </a-dropdown>
 </template>
 
 <script>
     import Dropdown from 'ant-design-vue/lib/dropdown/index.js';
-    import MenuItem from 'ant-design-vue/lib/menu/MenuItem.js';
+    import Tooltip  from 'ant-design-vue/lib/tooltip/index.js';
+    import MaButton from '../MaButton/MaButton.vue';
+    import MaIcon from '../MaIcon/MaIcon.vue';
+    const { Menu, MenuItem } = Dropdown;
     export default {
         name: 'ma-dropdown',
         components: {
+            MaButton,
+            MaIcon,
+            ATooltip: Tooltip,
             ADropdown: Dropdown,
             AMenuItem: MenuItem,
+            AMenu: Menu,
         },
         props: {
             menuItems: { type: Array, default: () => [] },
+            menuItemButton: { type: Object, default: null },
             title: { type: String, default: null },
-            content: { type: String, default: null },
+            trigger: { type: Array, default: () => ['hover'] },
```

Comment:
Click is more useful default variable on dashboard. 
```suggestion
            trigger: { type: Array, default: () => ['click'] },
```